src/ctap/test_helpers/env.rs

```diff
@@ -0,0 +1,49 @@
+use crate::clock::CtapInstant;
+use crate::ctap::command::{
+    AuthenticatorAttestationMaterial, AuthenticatorConfigParameters,
+    AuthenticatorVendorConfigureParameters, Command,
+};
+use crate::ctap::data_formats::ConfigSubCommand;
+use crate::ctap::status_code::Ctap2StatusCode;
+use crate::ctap::{key_material, Channel, CtapState};
+use crate::env::Env;
+
+// In tests where we define a dummy user-presence check that immediately returns, the channel
+// ID is irrelevant, so we pass this (dummy but valid) value.
+const DUMMY_CHANNEL: Channel = Channel::MainHid([0x12, 0x34, 0x56, 0x78]);
+
+pub struct SetupEnterpriseAttestationResponse {
+    pub attestation_material: AuthenticatorAttestationMaterial,
+}
+
+pub fn setup_enterprise_attestation(
+    state: &mut CtapState,
+    env: &mut impl Env,
+) -> Result<SetupEnterpriseAttestationResponse, Ctap2StatusCode> {
+    let config_params = AuthenticatorConfigParameters {
+        sub_command: ConfigSubCommand::EnableEnterpriseAttestation,
+        sub_command_params: None,
+        pin_uv_auth_param: None,
+        pin_uv_auth_protocol: None,
```

Comment:
I'm surprised this works without `pin_uv_auth_param` and `pin_uv_auth_protocol`?

Reply:
No pin was setup so it won't check those params. But we might need to change that after we add setup_pin helpers.

Reply:
Ah sorry, I misread the [specification](https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-20210615.html#authenticatorConfig) when double checking.

The way it's written is quite non-intuitive, it's basic structure is:
> 3. if not alwaysUv etc: goto 5
> 4. if PIN is set or alwaysUv: check PIN
> 5. process command

So what you do works until we also customize or configure alwaysUv!

Reply:
Then the current logic needs to change? We have
```
if storage::pin_hash(env)?.is_some() || enforce_uv {
```
meaning that we'll check it if pin is set or alwaysUv.

Reply:
No I think it's fine, my list was shortened for the explanation. You can double check the specification though if you feel like it.